extensions/ONNX/Onnx/OnnxConfig.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable IDE0130 // reduce number of "using" statements
+
+namespace Microsoft.KernelMemory;
+
+public class OnnxConfig
+{
+    /// <summary>
+    /// Path to the directory containing the .ONNX file.
+    /// </summary>
+    public string ModelPath { get; set; } = string.Empty;
```

Comment:
Do you think we should allow using different models for text generation and embeddings? 

For instance, I would like to use E5 for embeddings and Phi3.5 for text. 

Thoughts?

Reply:
see for example: https://github.com/microsoft/kernel-memory/blob/main/extensions/Ollama/Ollama/OllamaConfig.cs